### PR TITLE
Gromacs 5.0.5-intel2015a easycofig

### PR DESCRIPTION
**Pull request by ingridbr**
Add easyconfig for Gromacs 5.0.5 with intel/2015a

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.5-intel-2015a-hybrid.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('CMake', '3.1.0'),
+    ('libxml2', '2.9.2')
+]
+
+dependencies = [('Boost', '1.57.0', '-Python-2.7.9')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.5-intel-2015a-hybrid.eb]: current easyconfig for GROMACS 5.0.4 is using Boost 1.58, so please bump?

File: easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.5-intel-2015a-hybrid.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('CMake', '3.1.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-5.0.5-intel-2015a-hybrid.eb]: use latest CMake 3.2.2 (like in GROMACS 5.0.4 easyconfig we already have in place)

**Comment by ingridbr** [comment]
Requested changes made.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/894561d86f7d12f94cca for a full test report.

**Comment by boegel** [comment]
@ingridbr: ignore the failed test report, it's just a download issue (direct download over FTP doesn't work on test system due to firewall); I'll submit a new test report

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e189256940132f92ea37 for a full test report.

**Comment by boegel** [comment]
Looking great, going in, thanks @ingridbr!

And welcome to the wonderful world of EasyBuild! ;-)

**Comment by ingridbr** [comment]
Great!! That one was easy :-)

**Pull request merged**